src/routes/userapp/components/detail-page-specific/useAddComment.tsx

```diff
@@ -5,14 +5,21 @@ import { Comment } from "../../../../types";
 
 const addComment = (storeId: string, newComment: Comment) => {
   // no such path in mock
-  return fetch(`/api/stores/${storeId}/commentadd`, {
-    method: "POST",
-    headers: {
-      Accept: "application/json",
-      "Content-Type": "application/json",
+  return fetch(
```

Comment:
Nie można wyciągnąć tej logiki wyżej? Tzn mieć zmienną przechowującą aktualny backendUrl i w tych wszystkich fetchach uderzać po prostu na "backendUrl + '/api/addcomment/...'?
Aktualnie, żeby zmienić adres serwera trzeba podmieniać tą wartość we wszystkich fetchach, co się mija z zasadą dont repeat yourself.

Reply:
Nazwy endpointów nie pokrywają się też z tym co jest na backendzie.
Musisz dopasować je do tego co jest tu:
http://zpibackend.fly.dev/swagger-ui.html
Jak potrzebujesz pomocy z tym, to możemy się zdzwonić i to razem zrobić, będzie pewnie szybciej - swagger nie zawsze jest czytelny.

Reply:
Podmienilam na poprawne, z rezerwacja i sprawdzaniem dostępności nie macie, przydałby się jeszcze jeden na totalny store-config.

Z adresem servera mialam zaćmienie, on jest dodawany automatycznie, więc jest wyniesiony już dawno wyżej